Doc/library/unittest.mock.rst

```diff
@@ -30,7 +30,7 @@ module and class level attributes within the scope of a test, along with
 some examples of how to use :class:`Mock`, :class:`MagicMock` and
 :func:`patch`.
 
-Mock is very easy to use and is designed for use with :mod:`unittest`. Mock
+Mock is designed for use with :mod:`unittest`. Mock
```

Comment:
I suggest combining the two sentences.  

    s/. Mock/, and/

Reply:
Done, though I removed the comma in the substitution since the part after "and" is not an independent clause. Thanks for the review! It's a little change, but it's good to make Python more approachable however we can.